### PR TITLE
Pb pip

### PR DESCRIPTION
**Pull request by bloodbare**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: nucliadb_models/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 pydantic==1.9.0
-protobuf
\ No newline at end of file
+protobuf
+nucliadb-ingest>=1.0.0
+nucliadb-protos>=0.0.29
```
Comment (lferran) [inline nucliadb_models/requirements.txt:4]: It feels wrong that nucliadb_models is depending on nucliadb_ingest and nucliadb_protos.

Comment (lferran) [reply nucliadb_models/requirements.txt:4]: ah I see, it's because of the serialization logic

Comment (lferran) [reply nucliadb_models/requirements.txt:4]: This should remove the models -> ingest dependency:

https://github.com/nuclia/nucliadb/pull/291

File: nucliadb/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
 uvicorn
-pydantic_argparse
\ No newline at end of file
+pydantic_argparse
+nucliadb-one>=1.0.0
```
Comment (jotare) [inline nucliadb/requirements.txt:3]: nucliadb has dependency with nucliadb_writer. Now is solved with nucliadb_one, but we want to be more explicit?

**Review by jotare — CHANGES_REQUESTED** [review]

**Pull request merged**